MultiServer.py

```diff
@@ -777,6 +782,15 @@ def collect_player(ctx: Context, team: int, slot: int):
         register_location_checks(ctx, team, source_player, location_ids, count_activity=False)
         update_checked_locations(ctx, team, source_player)
 
+    if not is_group:
+        for group, group_players in ctx.groups.items():
+            if slot in group_players:
+                group_collected_players = ctx.group_collected.get(group, set())
```

Comment:
instead of a .get() follow by = you can also use .setdefault(), so 
`group_collected_players = ctx.group_collected.setdefault(group, set())`
allowing you to then no longer need the `ctx.group_collected[group] = group_collected_players` line later